tasks/user_accounts.yml

```diff
@@ -45,3 +45,21 @@
     createhome: False
   with_flattened:
     - '{{ sys_accs_cond | default([]) | difference(os_ignore_users) | list }}'
+
+- name: get all home directories in /home, but skip ignored users
+  find:
+    paths: /home/
+    recurse: no
+    file_type: directory
+    excludes: "{{ os_ignore_home_folder_users | join(',') }}"
+  register: home_directories
+  when: os_chmod_home_folders | bool
+
+- name: set ownership of /home directories to 0700
+  file:
+    mode: 0700
+    path: {{ item }}
+  loop:
+    - "{{ home_directories.files | map(attribute='path') | list }}"
+  ignore_errors: True
```

Comment:
Last one: ignore errors should be gone.

Reply:
Done. This was necessary for my first implementation, but not anymore with your method.